net-analyzer/graphite-web/graphite-web-1.1.3-r1.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1 python-utils-r1 prefix
+
+DESCRIPTION="Enterprise scalable realtime graphing"
+HOMEPAGE="https://graphiteapp.org/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+carbon ldap mysql memcached postgres +sqlite"
+
+DEPEND=""
+RDEPEND="
+	carbon? ( dev-python/carbon[${PYTHON_USEDEP}] )
+	ldap? ( dev-python/python-ldap[${PYTHON_USEDEP}] )
+	memcached? ( dev-python/python-memcached[${PYTHON_USEDEP}] )
+	mysql? (
+		|| (
+			dev-python/mysql-python[${PYTHON_USEDEP}]
+			dev-python/mysqlclient[${PYTHON_USEDEP}]
+		)
+	)
+	postgres? ( dev-python/psycopg:2[${PYTHON_USEDEP}] )
+	>=dev-python/django-1.8[sqlite?,${PYTHON_USEDEP}]
+	<dev-python/django-1.11.99[sqlite?,${PYTHON_USEDEP}]
+	>=dev-python/django-tagging-0.4.3[${PYTHON_USEDEP}]
+	dev-python/cairocffi[${PYTHON_USEDEP}]
+	dev-python/pyparsing[${PYTHON_USEDEP}]
+	dev-python/pytz[${PYTHON_USEDEP}]
+	dev-python/scandir[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/urllib3[${PYTHON_USEDEP}]
+	media-libs/fontconfig
+"
+
+PATCHES=(
+	# Do not install the configuration and data files. We install them
+	# somewhere sensible by hand.
+	"${FILESDIR}"/${PN}-1.1.3-fhs-paths.patch
+)
+
+python_prepare_all() {
+	# use FHS-style paths
+	export GRAPHITE_NO_PREFIX=yes
+	distutils-r1_python_prepare_all
+	eprefixify \
+		conf/graphite.wsgi.example \
+		webapp/graphite/local_settings.py.example
+}
+
+python_install_all() {
+	distutils-r1_python_install_all
+	keepdir /var/{lib,log}/${PN}
+	docinto examples
+	docompress -x "/usr/share/doc/${PF}/examples"
+	dodoc \
+		examples/example-graphite-vhost.conf \
+		conf/dashboard.conf.example \
+		conf/graphite.wsgi.example
+}
+
+python_install() {
+	distutils-r1_python_install \
+		--install-data="${EPREFIX}"/usr/share/${PN}
+
+	insinto /etc/${PN}
+	newins webapp/graphite/local_settings.py.example local_settings.py
+	pushd "${D}/$(python_get_sitedir)"/graphite > /dev/null || die
+	ln -s ../../../../../etc/${PN}/local_settings.py local_settings.py || die
+	popd > /dev/null || die
+}
+
+pkg_config() {
+	"${EROOT}"/usr/bin/django-admin.py migrate \
+		--settings=graphite.settings --run-syncdb
+	"${EROOT}"/usr/bin/build-index
```

Comment:
this name is way too generic, please rename it to ${PN}-build-index and fix the mentionings it elsewhere

Reply:
You're right, I'll update the patch to install just this script and not the other 2:
```
/usr/bin/run-graphite-devel-server.py
/usr/bin/build-index.sh
/usr/bin/build-index
```
And I will rename it.

The revbump is here indeed :)

Reply:
thanks for the work, by the way, it's a tremendously complex package to unravel 👍

Reply:
You're welcome! It took a few hours indeed. Too bad we both worked on it.

I was waiting for python-ldap to support Python 3 (which required quite a bit of work as well) but I didn't know where I could reach you since you're not on IRC.
What is the best way to contact you? Email?